docs/admin/admission-controllers.md

```diff
@@ -257,6 +257,14 @@ storage classes and how to mark a storage class as default.
 This plug-in sets the default forgiveness toleration for pods, which have no forgiveness tolerations, to tolerate
 the taints `notready:NoExecute` and `unreachable:NoExecute` for 5 minutes.
 
+### PodSecurityPolicy
```

Comment:
@pweil- Does it require PSP API (`--runtime-config=extensions/v1beta1/podsecuritypolicy=true`)? Should we mention it here?